### PR TITLE
Define configure default answers as the previously recorded answer

### PR DESCRIPTION
**Pull request by WilliamMcCumstie**
Continuation of Typology questions. Please merge #91 first

When loading the default answer, the answer file is checked first. If an old answer is found, then that is used as the default. Otherwise the default in `configure.yaml` is used.

### CODE REVIEW

File: spec/configurator_spec.rb [hunk 1]
```diff
@@ -282,5 +292,38 @@ def configure_with_input(input_string)
...
+            type: 'integer',
+            default: 10
+          }
+        }
+      })
+
+      old_answers = {
+        "string_q" => "CORRECT",
+        "integer_q" => -100,
+        "should_not_see_me" => "OHHH SNAP"
```
Comment (BobWhitelock) [inline spec/configurator_spec.rb]: :shipit:

**Comment by BobWhitelock** [comment]
Tests are failing for me on this branch. I thought it might be because of my earlier rebase, but they're also failing on your version at https://github.com/alces-software/metalware/tree/feature/rerun-configure-OLD. Note that branch also has an issue as there's a diff marker left in https://github.com/alces-software/metalware/commit/167a4f42767d8ddebc735e2ae5e922bcb010173d, which was a reason I rebased it, so it may be simpler to resolve this off of this branch.

**Comment by WilliamMcCumstie** [comment]
Seems to be an issue with one of the many rebases. The diff marker has been fixed in another rebase. All the test should be passing now (again).

**Comment by bobwhitelock** [comment]
LGTM, just did a little bit of refactoring before merging.

**Pull request merged**